R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
```

Comment:
The alternative is to do what you see here: https://github.com/tidyverse/glue/issues/7. That's the issue where, one day, glue might actually offer a replacement for our DIY versions.